plotly/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.16'
+__version__ = '2.0.17'
```

Comment:
I think we've actually been doing semver wrong. `Table` is a new feature, so I believe that this should be `2.1.0` (http://semver.org/)

Reply:
oh I thought we were only bumping if it was a new plotly.py specific feature